ci/travis/ci.sh

```diff
@@ -288,8 +288,7 @@ build_wheels() {
       # This command should be kept in sync with ray/python/README-building-wheels.md,
       # except the "${MOUNT_BAZEL_CACHE[@]}" part.
       # TODO(ilr) Re-add "suppress_output" to show info
-      docker run --rm -w /ray -v "${PWD}":/ray "${MOUNT_BAZEL_CACHE[@]}" \
-        -e TRAVIS_COMMIT="${TRAVIS_COMMIT}"  -e CI="${CI}" \
```

Comment:
these env vars are already included in the `MOUNT_BAZEL_CACHE` above

Reply:
LGTM!